test/docker-compose.yml

```diff
@@ -1,59 +1,42 @@
 # Test runner
-version: '3.4'
+version: '3.5'
 
-services:
+name: wikibase-suite-test-runner
+
+networks:
+  default:
+    name: wikibase-suite-test-services_default
 
+services:
   test-runner:
+    scale: 0
     build:
       context: ../test
       dockerfile: Dockerfile
+    stop_grace_period: 1s
     entrypoint: /bin/bash
     volumes:
-      - .:/usr/src/app
-      - node_modules:/usr/src/app/node_modules
+      - ..:/usr/src
+      - node_modules:/usr/src/test/node_modules
       - /var/run/docker.sock:/var/run/docker.sock
+    env_file: ./test-runner.env
```

Comment:
This makes all env vars in this file directly available in the test-runner container.